xapian-applications/omega/postprocess

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/python3
```

Comment:
This isn't always installed (eg on macOS). `/usr/bin/python` we may be able to assume, or `/usr/bin/env python` might be better. This however means you need to write code that's compatible with both py2 & py3. (It's almost, but not quite, acceptable any more to do py2 only. Big projects are starting the discussion of moving the default over to py3, and some smaller ones already have.)